src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -144,12 +144,12 @@ public ActionResult<IEnumerable<Course.Membership>> GetCourseMemberships(Guid id
             }
         }
         [HttpDelete]
-        [Route("{id}/DeleteCourseMembership")]
-        public ActionResult<Course> DeleteCourseMembership(Guid id, Guid userId)
+        [Route("{courseId}/DeleteCourseMembership")]
+        public ActionResult<Course> DeleteCourseMembership(Guid courseId, Guid userId, bool includeMemebrship, bool includeUser)
```

Comment:
You don't need to expose include in the API. You know that the memberships should be fetched and can pass true to te service.

Reply:
Deleted includes and changed id > courseId